outgoingra/src/main/java/no/bankid/outgoing/ra/SendActivationCodeRequestBody.java

```diff
@@ -0,0 +1,23 @@
+package no.bankid.outgoing.ra;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.UUID;
+
+@Schema(description = "Request body content for asking an RA to send an activation code to an endUser")
+public class SendActivationCodeRequestBody extends AuthenticationBody {
+
+    public enum ActivationCodeType { digits, words }
+
+    @Schema(description = "The id of this activation attempt, used for for logging, will be the same for both codes")
+    public UUID activation_id;
+    @Schema(description = "One of the method identifiers returned from check_user_selfservice call")
+    public String distribution_method;
+    public ActivationCodeLocale locale;
+    @Schema(description = "Whether the activation code is a number or two code words")
```

Comment:
No longer applicable, drop

Reply:
Dropped SendActivationCodeRequestBody and the corr. SendActivationCodeErrorResponse.